pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -208,6 +208,9 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 		doc.Dequeues = 0
 	}
 
+	// TODO: Remove this once 23-04-01 API release is complete.
+	determineOutboundType(ctx, doc, subscription)
+
 	// SetDefaults will set defaults on cluster document
 	api.SetDefaults(doc)
```

Comment:
does api.SetDefaults still have the default setting for UDR?  If so, we're good as it will cover the existing cluster PUCMs from a frontend perspective.

Reply:
Yes, api.SetDefaults will set it to Loadbalancer